Source/WebKit/UIProcess/RemoteLayerTree/mac/RemoteLayerTreeDrawingAreaProxyMac.mm

```diff
@@ -136,6 +136,9 @@ explicit RemoteLayerTreeDisplayLinkClient(WebPageProxyIdentifier pageID)
         removeTransientZoomFromLayer();
         m_transactionIDAfterEndingTransientZoom = { };
     }
+    
+    if (m_transactionIDForSentTransientZoomUpdate && transaction.transactionID() >= m_transactionIDForSentTransientZoomUpdate)
+        m_transactionIDForSentTransientZoomUpdate = { };
```

Comment:
We should clear m_transactionIDForSentTransientZoomUpdate in `commitTransientZoom()` I think too?

Reply:
Fixed.